packages/s3-presigned-post/package.json

```
@@ -9,7 +9,8 @@
     "build:types": "tsc -p tsconfig.types.json",
     "build:types:downlevel": "downlevel-dts dist-types dist-types/ts3.4",
     "clean": "rimraf ./dist-* && rimraf *.tsbuildinfo",
-    "test": "jest"
+    "test": "jest",
+    "test:integration": "jest -c jest.integ.config.js"
```

Change this to `test:e2e` so that it's run from tests/e2e: https://github.com/aws/aws-sdk-js-v3/blob/main/tests/e2e/index.js

```suggestion
    "test:e2e": "jest -c jest.integ.config.js"
```

Let's stick to `.integ.spec.ts`
We need to change `.ispec.ts` files to `.integ.spec.ts`